web/templates/base.html

```diff
@@ -1,4 +1,5 @@
 {% load i18n static libravatar %}
+{% load humanize %}
```

Comment:
tum loadlari ayni satirda yapa biliyorsun, bir onceki load-in icine eklemen yeterli olur